web/docs/Backend Starters/Decoupled Drupal/creating-new-project.md

```diff
@@ -1,25 +1,167 @@
+
+- Pantheon repo is your source of truth
+
+### Installing using Dashboard Upstream
+
+- Create an empty upstream site one of two ways:
+
+  - Via the Pantheon Dashboard at this link:
+
+    - [Empty Upstream](https://dashboard.pantheon.io/sites/create?upstream_id=4c7176de-e079-eed1-154d-44d5a9945b65)
+
+  - Or Alternatively via Terminus:
+
+    ```
+    terminus site:create my-new-site "Describe Site" empty --org='My Team Name'
+    ```
+
+  - Replace `'{My Team Name}'` with your team name - for example `My Agency`. This can also be omitted.
+
+#### In the Pantheon Admin Dashboard:
+
+Switch **Development Mode** to **Git**
+
+Click the **Upgrade** button, and change the site plan to **Performance** > **Medium**.
+
+\*\* Clone your site's repository on your local
+
+\*\* Change into the newly cloned directory
+
+\*\* Add remote for the decoupled upstream: `git remote add upstream https://github.com/pantheon-systems/decoupled-drupal-composer-managed`
+
+\*\* Checkout the main branch from upstream:
+
+```
+
+git fetch upstream main
+
+git checkout -t upstream/main
+
+```
+
+\*\* Reset the master branch to match main, then force push that to the Dev environment:
+
+```
+
+git branch -D master
+
+git checkout -b master
+
+git push origin master --force
+
+```
+
+:::Note
+Make sure code has synced and all active workflows are complete in the UI.
+Click on the **Visit Development Site** button to Install via the UI—selecting either the Pantheon Decoupled Profile, or Pantheon Decoupled Umami Demo profiles.
+:::
+
+## Installing using Build Tools
+
+### Prerequisites
 
 - Composer (required for CMS backends): [Install Globally](https://getcomposer.org/download/)
 - [Generate machine token](https://pantheon.io/docs/machine-tokens#create-a-machine-token) & [Authenticate into Terminus](https://pantheon.io/docs/machine-tokens#authenticate-into-terminus)
 - [Install Terminus](https://pantheon.io/docs/terminus/install) (3.0.0 above required)
 - Also install the following plugins:
   - `terminus self:plugin:install terminus-build-tools-plugin`
-  - `terminus self:plugin:install terminus-power-tools`
+  - `terminus self:plugin:install terminus-power-tools` [Known Issues](#known-issues)
   - `terminus self:plugin:install terminus-secrets-plugin`
   - Reload the terminus plugins: `terminus self:plugin:reload`
   - Clear cache for composer: `composer clear-cache`
   - Validate that the required plugins are installed: `terminus self:plugin:list`
 - Create [Github Personal access tokens](https://github.com/settings/tokens)
-- Create [CircleCI Personal API Tokens](https://app.circleci.com/settings/user/tokens)
+- [For other providers](#additional-tokens)
+- Build Tools should prompt the user for the appropriate tokens during [Installation](#installation)
 
-## Installation
+#### Known Issues
+
+- If you encounter errors during the [Installation](#installation) process below. You may remove the terminus-power-tools plugin and running the install process again.
```

Comment:
Could you move the known issues section to the bottom? Likely the last section before the deprecated heading.

This also reads as two incomplete statements, and should be combined into a single sentence.